sockeye/inference.py

```diff
@@ -1715,32 +1850,43 @@ def _get_best_from_beam(self,
         :param beam_histories: The beam histories for each sentence in the batch.
         :return: List of Translation objects containing all relevant information.
         """
-        # Initialize the best_ids to the first item in each batch
-        best_ids = np.arange(0, self.batch_size * self.beam_size, self.beam_size, dtype='int32')
+        nbest_translations = []  # type: List[List[Translation]]
```

Comment:
the doc-string above should maybe be adjusted, saying that the top-n translations are returned.

Reply:
done